### PR TITLE
Changed master service to master-api and master-controllers

### PR DESCRIPTION
**Pull request by mburke5678**
Replace `atomic-openshift-master` with `atomic-openshift-master-api atomic-openshift-master-controllers` and `origin-master` with `origin-master-api` and `origin-master-controllers`

https://bugzilla.redhat.com/show_bug.cgi?id=1537756

### CODE REVIEW

**Comment by mburke5678** [comment]
@sdodson Can you take a look at the changes I made?

File: install_config/downgrade.adoc [hunk 1]
```diff
@@ -57,6 +57,8 @@ regardless:
 
 ----
 /etc/sysconfig/atomic-openshift-master
+/etc/atomic-openshift-master-api
+/etc/atomic-openshift-master-controller
```
Comment (sdodson) [inline install_config/downgrade.adoc]: These are in /etc/sysconfig too
```
/etc/sysconfig/atomic-openshift-master-api
/etc/sysconfig/atomic-openshift-master-controller
```

File: install_config/configuring_aws.adoc [hunk 2]
```diff
@@ -272,9 +274,10 @@ key `kubernetes.io/cluster/<name>,Value=<clusterid>` and have the same value for
 $ oc get pv -o json|jq '.items[].spec.awsElasticBlockStore.volumeID'
 ----
 
-. Restart `atomic-openshift-master` and `atomic-openshift-node` on all nodes:
+. Restart the master service on the master and the node service on all nodes:
```
Comment (sdodson) [inline install_config/configuring_aws.adoc]: s/master service/master services/

**Review by sdodson — CHANGES_REQUESTED** [review]
Looks good, the noted sysconfig path is the only real problem I see.

**Comment by mburke5678** [comment]
@jianlinliu PTAL

File: install_config/upgrading/migrating_etcd.adoc [hunk 3]
```diff
@@ -111,9 +111,7 @@ for steps.
 .. Stop all master services:
 +
 ----
-# systemctl stop atomic-openshift-master \
-    atomic-openshift-master-api \
-    atomic-openshift-master-controllers
+# systemctl restart atomic-openshift-master-api atomic-openshift-master-controllers
```
Comment (jianlinliu) [inline install_config/upgrading/migrating_etcd.adoc]: here should be "stop", but not "restart", right?

File: install_config/upgrading/migrating_etcd.adoc [hunk 4]
```diff
@@ -286,9 +277,7 @@ from a backup:
 . Stop the master services:
 +
 ----
-# systemctl stop atomic-openshift-master \
-    atomic-openshift-master-api \
-    atomic-openshift-master-controllers
+# systemctl restart atomic-openshift-master-api atomic-openshift-master-controllers
```
Comment (jianlinliu) [inline install_config/upgrading/migrating_etcd.adoc]: Here should be "stop", but not "restart", if I am right.

File: install_config/upgrading/migrating_etcd.adoc [hunk 5]
```diff
@@ -315,12 +304,11 @@ or xref:../../admin_guide/backup_restore.adoc#cluster-restore-single-member-etcd
...
-# systemctl start atomic-openshift-master
+# systemctl restart atomic-openshift-master-api atomic-openshift-master-controllers
 ----
 +
 For multiple master clusters, run the following on all masters:
 +
 ----
-# systemctl start atomic-openshift-master-api \
-    atomic-openshift-master-controllers
+# systemctl restart atomic-openshift-master-api atomic-openshift-master-controllers
```
Comment (jianlinliu) [inline install_config/upgrading/migrating_etcd.adoc]: For single maser or multiple master clusters, the same command is executed, they should be merged together, just like what you did in the above modification.

File: install_config/upgrading/migrating_etcd.adoc (hunk 4 above)
Comment (mburke5678) [reply install_config/upgrading/migrating_etcd.adoc]: @jianlinliu Thank you! Thank you! Copy/paste error.

Comment (mburke5678) [reply install_config/upgrading/migrating_etcd.adoc]: @jianlinliu Thank you! Thank you! Copy/paste error.

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL

**Comment by mburke5678** [comment]
[rev_history]
|xref:../welcome/index.html#welcome-index[OpenShift Container Platform 3.5 Documentation]
|Replace `atomic-openshift-master` with `atomic-openshift-master-api atomic-openshift-master-controllers` and `origin-master` with `origin-master-api` and `origin-master-controllers` throughout the documents.
%

**Review by sdodson — APPROVED** [review]

File: install_config/web_console_customization.adoc [hunk 6]
```diff
@@ -991,16 +972,54 @@ assetConfig:
...
-# systemctl restart atomic-openshift-master
+# systemctl restart atomic-openshift-master-api atomic-openshift-master-controllers
 endif::[]
 ----
 
 xref:../install_config/router/default_haproxy_router.adoc#using-wildcard-routes[Learn
 how to configure HAProxy routers to allow wildcard routes].
 
+[[web-console-enable-tech-preview-feature]]
+== Enabling Features in Technology Preview
```
Comment (bfallonf) [inline install_config/web_console_customization.adoc]: This bit as about the web console specifically, correct? Might be better to call that out: 

Enabling Technology Previews features to show in the web console

File: install_config/web_console_customization.adoc [hunk 7]
```diff
@@ -991,16 +972,54 @@ assetConfig:
...
+Currently, there are no web console features in Technology Preview.
+
+To enable a Technology Preview feature:
+
+. Save this script to a file (for example, *_tech-preview.js_*):
++
+[source, javascript]
+----
+window.OPENSHIFT_CONSTANTS.ENABLE_TECH_PREVIEW_FEATURE.<feature_name> = true;
+----
```
Comment (bfallonf) [inline install_config/web_console_customization.adoc]: The above confused me a little. Is there reader creating the tech-preview.js file, then adding above line to it? Is that all? And how do they know what the feature name is for the feature?

**Comment by bfallonf** [comment]
@mburke5678 largely all good, but I made some comments in the web console tech preview section. I may have given you more questions...

**Comment by jianlinliu** [comment]
LGTM.

File: install_config/web_console_customization.adoc (hunk 7 above)
Comment (mburke5678) [reply install_config/web_console_customization.adoc]: @bfallonf This PR was specific to changing the master service names. I can open a follow-up to persue these matters.

File: install_config/web_console_customization.adoc (hunk 6 above)
Comment (mburke5678) [reply install_config/web_console_customization.adoc]: Follow-up in https://github.com/openshift/openshift-docs/pull/7528

**Comment by mburke5678** [comment]
@sdodson I got a comment from @jiajliu in QE that this change is for 3.7+ and I merged this change to 3.6.  Can you confirm if the `atomic-openshift-master` was unmasked in 3.6 or not? 

https://bugzilla.redhat.com/show_bug.cgi?id=1548556#c4
"From v3.7, only "atomic-openshift-master-api.service and atomic-openshift-master-controllers.service" for any deployment.
3.6+ should mean >v3.6 in https://bugzilla.redhat.com/show_bug.cgi?id=1537756#c2"

**Pull request merged**